tests/conftest.py

```diff
@@ -110,6 +110,7 @@ def f(self2, reg):
     def call_xfail_no_output(self, cmd):
         proc = self.unchecked_call(cmd)
         assert proc.returncode != 0
+        assert proc.stderr == ""
```

Comment:
Didn't we, some time in the past, also assert that stdout is empty in an xfail call? That doesn't seem to be the case anymore. Not sure if we dropped it on purpose, though.

In any case, it's probably better to call `call_xfail` here instead of `unchecked_call`, since it is a specialization of `call_xfail`. Cleaning that up might well be done separately, so this PR looks good to me!